### PR TITLE
Update README config settings

### LINKED ISSUE #722: Lost default configuration for OTLP exporter endpoint
Upon further digging, it is found out that endpoint to otel exporter to be configured via env var else results in the following error. 

Stacktrace for reference
`[opentelemetry.auto.trace 2020-07-17 17:27:26:373 +0530] [main] ERROR io.opentelemetry.auto.bootstrap.Agent - Throwable thrown while installing the agent tracer
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at io.opentelemetry.auto.bootstrap.Agent.installAgentTracer(Agent.java:175)
	at io.opentelemetry.auto.bootstrap.Agent.start(Agent.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at io.opentelemetry.auto.bootstrap.AgentBootstrap.agentmain(AgentBootstrap.java:73)
	at io.opentelemetry.auto.bootstrap.AgentBootstrap.premain(AgentBootstrap.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:386)
	at sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:401)
Caused by: java.lang.NullPointerException: channel
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:910)
	at io.opentelemetry.auto.shaded.io.grpc.stub.AbstractStub.<init>(AbstractStub.java:73)
	at io.opentelemetry.auto.shaded.io.grpc.stub.AbstractBlockingStub.<init>(AbstractBlockingStub.java:39)
	at io.opentelemetry.proto.collector.trace.v1.TraceServiceGrpc$TraceServiceBlockingStub.<init>(TraceServiceGrpc.java:185)
	at io.opentelemetry.proto.collector.trace.v1.TraceServiceGrpc$TraceServiceBlockingStub.<init>(TraceServiceGrpc.java:182)
	at io.opentelemetry.proto.collector.trace.v1.TraceServiceGrpc$2.newStub(TraceServiceGrpc.java:89)
	at io.opentelemetry.proto.collector.trace.v1.TraceServiceGrpc$2.newStub(TraceServiceGrpc.java:86)
	at io.opentelemetry.auto.shaded.io.grpc.stub.AbstractBlockingStub.newStub(AbstractBlockingStub.java:64)
	at io.opentelemetry.auto.shaded.io.grpc.stub.AbstractBlockingStub.newStub(AbstractBlockingStub.java:51)
	at io.opentelemetry.proto.collector.trace.v1.TraceServiceGrpc.newBlockingStub(TraceServiceGrpc.java:92)
	at io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter.<init>(OtlpGrpcSpanExporter.java:87)
	at io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter.<init>(OtlpGrpcSpanExporter.java:71)
	at io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter$Builder.build(OtlpGrpcSpanExporter.java:259)
	at io.opentelemetry.auto.exporters.otlp.OtlpSpanExporterFactory.fromConfig(OtlpSpanExporterFactory.java:31)
	at io.opentelemetry.auto.tooling.TracerInstaller.installExporter(TracerInstaller.java:133)
	at io.opentelemetry.auto.tooling.TracerInstaller.installExporters(TracerInstaller.java:65)
	at io.opentelemetry.auto.tooling.TracerInstaller.installAgentTracer(TracerInstaller.java:52)
	... 18 more`
Checked in [opentelemetry-java otel exporter](https://github.com/open-telemetry/opentelemetry-java/blob/v0.6.0/exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcSpanExporter.java#L171). There is no default managed channel pointing to the otel exporter endpoint. 

Probably, Can't we can change this documentation to make the endpoint setting as mandatory.

> By default OpenTelemetry Java agent uses OTLP exporter configured to send data to OpenTelemetry collector at localhost:55680.

### PR DESCRIPTION
**Pull request by trask**
I'm going to leave this as a draft for now, I think probably should wait until (right before or right after) the 0.7.0 release to merge this.

Closes #722

### CODE REVIEW

**Comment by trask** [comment]
I added a commit to convert `ota` to `otel` in anticipation of #841 being merged prior to 0.7.0 release

File: README.md [hunk 1]
```diff
@@ -105,9 +105,12 @@ attributes to stdout. It is used mainly for testing and debugging.
 
 The propagator controls which distributed tracing header format is used.
 
-| System property  | Environment variable | Purpose                                                                                 |
-|------------------|----------------------|-----------------------------------------------------------------------------------------|
-| otel.propagators | OTEL_PROPAGATORS     | Default is "tracecontext" (W3C). Other supported values are "b3", "b3single", "jaeger". |
+If this is set to a comma-delimited list of the values, the multi-propagator will be used.
+[TODO link to specification that explains multi-propagator behavior, see [tracking issue](https://github.com/open-telemetry/opentelemetry-specification/issues/496) for now]
```
Comment (iNikem) [inline README.md:109]: I vaguely remember, that otel-java has their javadoc published somewhere. We can reference that.

Comment (trask) [reply README.md]: I found [the javadoc](https://javadoc.io/doc/io.opentelemetry/opentelemetry-extension-trace-propagators/latest/io/opentelemetry/extensions/trace/propagation/TraceMultiPropagator.html), but then decided just to include brief explanation here inline

File: README.md [hunk 2]
```diff
@@ -96,27 +96,31 @@ to the OpenTelemetry Resource ([see below](#opentelemetry-resource)), e.g. `OTEL
...
 
 #### Propagator
 
 The propagator controls which distributed tracing header format is used.
 
-| System property | Environment variable | Purpose                                                                                 |
-|-----------------|----------------------|-----------------------------------------------------------------------------------------|
-| ota.propagators | OTA_PROPAGATORS      | Default is "tracecontext" (W3C). Other supported values are "b3", "b3single", "jaeger". |
+If this is set to a comma-delimited list of the values, the multi-propagator will be used.
+The multi-propagator will try to extract the context from incoming requests using each of the configured propagator formats (in order), stopping after the first successful context extraction.
```
Comment (iNikem) [inline README.md:109]: Maybe just reference the javadoc of that class?

Comment (trask) [reply README.md:109]: I thought the text description was nice, and doesn't scare away Ops folks who are not java developers 😄

**Pull request merged**